src/views/confirm-registered-office-address.njk

```diff
@@ -49,7 +49,7 @@
             <br>
           </p>
           <p class="govuk-body-m">
-            <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterManualAddressPageType) }}">{{ i18n.confirmRegisteredOfficeAddressPage.addressLink }} <span class="govuk-visually-hidden">{{ i18n.confirmRegisteredOfficeAddressPage.addressLinkHidden }}</span></a>
+            <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterManualAddressPageType) }}" data-event-id="change-link">{{ i18n.confirmRegisteredOfficeAddressPage.addressLink }} <span class="govuk-visually-hidden" data-event-id="change">{{ i18n.confirmRegisteredOfficeAddressPage.addressLinkHidden }}</span></a>
```

Comment:
for the 2nd data-event-id (data-event-id="change") it looks like it is for a hidden change link ( not sure how that works tbh) , i would say change the name of that event id to 'change-link-hidden'